GameServer/ECS-Components/AttackComponent.cs

```diff
@@ -2174,13 +2172,6 @@ public virtual eAttackResult CalculateEnemyAttackResult(WeaponAction action, Att
             if (grapple != null)
                 return eAttackResult.Grappled;
 
-            if (brittleguard != null)
```

Comment:
this is all handled via a 100% intercept rate on the pet. no need for a specific check here